draft-ietf-sidrops-aspa-verification.xml

```diff
@@ -346,153 +426,232 @@ authorized(AS x, AS y) =  / Else, "Provider+" if the VAP-SPAS entry
     </section>
-
-    <section title="Logging" anchor="logging">
-      <t>
-	    For any route with an Invalid AS_PATH, the cause of the Invalid state SHOULD be logged for monitoring and diagnostic purposes.
-	    The cause of the Invalid state can be in the form of listing the AS hops which were evaluated by the provider authorization function to be &quot;Not Provider+&quot;.
-	    The logging router, however, cannot necessarily determine the origin of the route leak.
-      </t>
+     <!--
+      The configured BGP Roles SHOULD be used to automate the use of described above verification procedures helping to distinguish whether upstream or downstream procedures should be applied.
+      [Sriram note: This can be expressed better which I did above.]
+     
+      In case of Complex peering relations that can't be segregated in multiple eBGP sessions, an operator may want to achieve an equivalent outcome by configuring policies with corresponding verification procedures on a per-prefix basis.
+      [Sriram note: ASPA are not per-prefix. So this is not possible.]
+     -->
+   <section title="Only to Customer (OTC) Attribute" anchor="otc">
+   <t>
+    While the ASPA-based AS_PATH verification method (<xref target="verif"/>, <xref target="impl-verif-mitig"/>) detects and mitigates route leaks that were created by preceding ASes listed in the AS_PATH, it lacks the ability to prevent the local AS from initiating a route leak towards its neighbor.
+    The use of the Only to Customer (OTC) Attribute fills in that gap (see Section 5, <xref target="RFC9234"/>).
+    The procedures utilizing the OTC Attribute set out in <xref target="RFC9234"/> complement the ASPA-based method.
+    Implementation of those procedures is RECOMMENDED.
+   </t>
+   </section>
+   </section>
+
+  <section title="Problems Addressed by ASPA and Early Adoption Benefits" anchor="property">
+    <t>
+      The ASPA-based path verification (<xref target="verif" />, <xref target="impl-verif-mitig" />) mainly addresses detection and mitigation of route leaks.
+      As a side benefit, protection is provided against some cases of forged-origin and forged-path-segment prefix hijacks (Properties 2 and 3 below).
+      Undoubtedly, there exist types of AS path manipulation attacks that are not aimed to be solved by ASPA (see <xref target="security"/>).
+    </t>
+    <t>
+      The ASPA method has the properties (i.e., anomaly detection capabilities) listed below.
+      Partial deployment scenarios and early adoption benefits are considered.
+      In the case of property 1, it is assumed that the attacks involve route leaks but not malicious removal of ASes with ASPA records from the AS path.
+    </t>
+    <t>
+      <list style="">
+        <t>
+          <strong>Property 1 (Route Leak Detection):</strong> Let AS A and AS B be any two ASes in the Internet doing ASPA (registration and path verification) and no assumption is made about the ASPA deployment status of other ASes.
```

Comment:
>Actually AS A only needs to do the regirstration while AS B only does the verification.

True. But it is easy to visualize that an AS doing registration would also do verification and vice versa. Also, if they both do registration and verification, the benefit is seen in both directions.

>So the corollary could be more clear about this.

It would be very welcome if you suggest some wording.